[2019-12-17 18:58:26,111] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/2 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = qry.get(id)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 982, in get
    return self._get_impl(ident, loading.load_on_pk_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 1094, in _get_impl
    return db_load_fn(self, primary_key_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/loading.py", line 221, in load_on_pk_identity
    q._get_condition()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 430, in _get_condition
    "get", order_by=False, distinct=False
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 455, in _no_criterion_condition
    self._no_criterion_assertion(meth, order_by, distinct)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 451, in _no_criterion_assertion
    "Query with existing criterion. " % meth
sqlalchemy.exc.InvalidRequestError: Query.get() being called on a Query with existing criterion. 
[2019-12-17 18:58:26,113] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 18:58:28,061] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/3 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = qry.get(id)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 982, in get
    return self._get_impl(ident, loading.load_on_pk_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 1094, in _get_impl
    return db_load_fn(self, primary_key_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/loading.py", line 221, in load_on_pk_identity
    q._get_condition()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 430, in _get_condition
    "get", order_by=False, distinct=False
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 455, in _no_criterion_condition
    self._no_criterion_assertion(meth, order_by, distinct)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 451, in _no_criterion_assertion
    "Query with existing criterion. " % meth
sqlalchemy.exc.InvalidRequestError: Query.get() being called on a Query with existing criterion. 
[2019-12-17 18:58:28,063] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 18:58:29,376] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/4 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = qry.get(id)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 982, in get
    return self._get_impl(ident, loading.load_on_pk_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 1094, in _get_impl
    return db_load_fn(self, primary_key_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/loading.py", line 221, in load_on_pk_identity
    q._get_condition()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 430, in _get_condition
    "get", order_by=False, distinct=False
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 455, in _no_criterion_condition
    self._no_criterion_assertion(meth, order_by, distinct)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 451, in _no_criterion_assertion
    "Query with existing criterion. " % meth
sqlalchemy.exc.InvalidRequestError: Query.get() being called on a Query with existing criterion. 
[2019-12-17 18:58:29,378] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 18:58:30,879] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": [{"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 1, "name": "Wildanzzz", "age": 22, "sex": "Male"}, {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 2, "name": "Wildan", "age": 22, "sex": "Male"}]}
[2019-12-17 18:59:20,777] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [DELETE]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 78, in delete
    qry = qry.get(id)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 982, in get
    return self._get_impl(ident, loading.load_on_pk_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 1094, in _get_impl
    return db_load_fn(self, primary_key_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/loading.py", line 221, in load_on_pk_identity
    q._get_condition()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 430, in _get_condition
    "get", order_by=False, distinct=False
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 455, in _no_criterion_condition
    self._no_criterion_assertion(meth, order_by, distinct)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 451, in _no_criterion_assertion
    "Query with existing criterion. " % meth
sqlalchemy.exc.InvalidRequestError: Query.get() being called on a Query with existing criterion. 
[2019-12-17 18:59:20,779] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "DELETE", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 18:59:28,279] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": [{"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 1, "name": "Wildanzzz", "age": 22, "sex": "Male"}, {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 2, "name": "Wildan", "age": 22, "sex": "Male"}]}
[2019-12-17 19:08:09,322] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": [{"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 1, "name": "Wildanzzz", "age": 22, "sex": "Male"}, {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 2, "name": "Wildan", "age": 22, "sex": "Male"}]}
[2019-12-17 19:08:12,394] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = Users.get(id).filter_by(deleted_status=False)
AttributeError: type object 'Users' has no attribute 'get'
[2019-12-17 19:08:12,395] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:08:27,181] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = Users.query.get(id).filter_by(deleted_status=False)
AttributeError: 'Users' object has no attribute 'filter_by'
[2019-12-17 19:08:27,184] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:08:50,315] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = Users.query.get(id).filter_by(deleted_status=False)
AttributeError: 'Users' object has no attribute 'filter_by'
[2019-12-17 19:08:50,317] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:09:46,040] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = Users.query.get(id).filter_by(deleted_status=False)
AttributeError: 'Users' object has no attribute 'filter_by'
[2019-12-17 19:09:46,042] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:10:15,367] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = Users.query.get(id).where(deleted_status=False)
AttributeError: 'Users' object has no attribute 'where'
[2019-12-17 19:10:15,370] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:10:42,196] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = Users.filter_by(deleted_status=False).get(id)
AttributeError: type object 'Users' has no attribute 'filter_by'
[2019-12-17 19:10:42,198] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:10:55,362] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = Users.query.filter_by(deleted_status=False).get(id)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 982, in get
    return self._get_impl(ident, loading.load_on_pk_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 1094, in _get_impl
    return db_load_fn(self, primary_key_identity)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/loading.py", line 221, in load_on_pk_identity
    q._get_condition()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 430, in _get_condition
    "get", order_by=False, distinct=False
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 455, in _no_criterion_condition
    self._no_criterion_assertion(meth, order_by, distinct)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 451, in _no_criterion_assertion
    "Query with existing criterion. " % meth
sqlalchemy.exc.InvalidRequestError: Query.get() being called on a Query with existing criterion. 
[2019-12-17 19:10:55,367] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:13:20,441] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = Users.query.id(id)
AttributeError: 'BaseQuery' object has no attribute 'id'
[2019-12-17 19:13:20,447] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:13:35,031] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 38, in get
    qry = Users.query(id)
TypeError: 'BaseQuery' object is not callable
[2019-12-17 19:13:35,036] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:14:09,270] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 39, in get
    if qry["deleted_status"] is True or qry is None:
TypeError: 'Users' object is not subscriptable
[2019-12-17 19:14:09,273] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:14:58,169] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 1, "name": "Wildanzzz", "age": 22, "sex": "Male"}}
[2019-12-17 19:17:14,721] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan", "age": 22, "sex": "Male"}, "response": {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 1, "name": "Wildanzzz", "age": 22, "sex": "Male"}}
[2019-12-17 19:17:27,133] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "PUT", "code": "404 NOT FOUND", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 19:19:45,129] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 1, "name": "Wildanzzz", "age": 22, "sex": "Male"}}
[2019-12-17 19:19:49,952] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 19:19:40 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male"}}
[2019-12-17 19:20:04,872] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "DELETE", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": {"message": "Deleted"}}
[2019-12-17 19:20:10,811] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 19:19:40 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male"}}
[2019-12-17 19:20:45,289] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": [{"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 19:19:40 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male"}, {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 2, "name": "Wildan", "age": 22, "sex": "Male"}]}
[2019-12-17 19:21:32,554] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": [{"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 19:19:40 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male"}, {"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 2, "name": "Wildan", "age": 22, "sex": "Male"}]}
[2019-12-17 19:21:40,549] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "DELETE", "code": "404 NOT FOUND", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 19:21:45,381] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "DELETE", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": {"message": "Deleted"}}
[2019-12-17 19:21:51,900] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "404 NOT FOUND", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 19:21:54,668] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": [{"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 2, "name": "Wildan", "age": 22, "sex": "Male"}]}
[2019-12-17 19:22:30,129] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "PUT", "code": "400 BAD REQUEST", "request": {}, "response": {"message": "The browser (or proxy) sent a request that this server could not understand."}}
[2019-12-17 19:22:48,546] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "PUT", "code": "400 BAD REQUEST", "request": {}, "response": {"message": "The browser (or proxy) sent a request that this server could not understand."}}
[2019-12-17 19:22:54,871] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "PUT", "code": "404 NOT FOUND", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "deleted_status": 0}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 19:23:27,464] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "404 NOT FOUND", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 19:23:30,519] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": [{"created_at": "Tue, 17 Dec 2019 18:56:42 -0000", "updated_at": "Tue, 17 Dec 2019 18:56:42 -0000", "id": 2, "name": "Wildan", "age": 22, "sex": "Male"}]}
[2019-12-17 19:44:16,066] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male"}, "response": []}
[2019-12-17 19:45:29,180] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 1, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:46:10,451] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:46:11,305] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:46:12,148] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:46:21,002] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1969, in finalize_request
    response = self.process_response(response)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 2266, in process_response
    response = handler(response)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py", line 36, in after_request
    "response": json.loads(response.data.decode("utf-8"))
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2019-12-17 19:46:21,004] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1975} ERROR - Request finalizing failed with an error while handling an error
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1969, in finalize_request
    response = self.process_response(response)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 2266, in process_response
    response = handler(response)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py", line 43, in after_request
    "response": json.loads(response.data.decode("utf-8"))
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[2019-12-17 19:46:24,134] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 1, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 19:47:01,679] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "PUT", "code": "404 NOT FOUND", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 19:47:04,934] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:48:51,438] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:49:19,967] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:49:29,354] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /client/1 [POST]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
TypeError: post() got an unexpected keyword argument 'id'
[2019-12-17 19:49:29,356] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"message": "Internal Server Error"}}
[2019-12-17 19:49:33,115] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:49:16 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 5, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:49:49,585] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": false}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": false}}
[2019-12-17 19:50:08,242] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:50:14,957] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:50:16,757] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:50:19,430] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}}
[2019-12-17 19:50:27,173] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": true}}
[2019-12-17 19:50:35,888] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}}
[2019-12-17 19:50:44,740] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}}
[2019-12-17 19:50:54,524] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "PUT", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}}
[2019-12-17 19:51:09,937] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "DELETE", "code": "404 NOT FOUND", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 19:51:20,034] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:49:16 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 5, "client_key": "CLIENTzzz", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 19:51:33,617] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "DELETE", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": {"message": "Deleted"}}
[2019-12-17 19:51:38,641] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "404 NOT FOUND", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 19:51:42,339] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:07:40,215] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:08:27,477] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}}
[2019-12-17 20:08:33,126] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:09:51,999] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:11:05,152] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}]}
[2019-12-17 20:11:19,195] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}]}
[2019-12-17 20:12:03,663] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}]}
[2019-12-17 20:12:39,922] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}]}
[2019-12-17 20:13:16,735] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:15:21,435] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /client [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/client/resources.py", line 25, in get
    if args["status"].lower() == "true":
AttributeError: 'NoneType' object has no attribute 'lower'
[2019-12-17 20:15:21,437] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": null, "response": {"message": "Internal Server Error"}}
[2019-12-17 20:16:05,049] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:16:19,687] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:16:26,423] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:16:32,317] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:17:10,586] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:49:16 -0000", "id": 1, "client_key": "CLIENTzzz", "client_secret": "zzzz", "status": false}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:17:14,530] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": []}
[2019-12-17 20:18:13,243] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user [POST]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 53, in post
    if args["client_id"] in Clients.query(Clients.id):
TypeError: 'BaseQuery' object is not callable
[2019-12-17 20:18:13,246] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 5}, "response": {"message": "Internal Server Error"}}
[2019-12-17 20:23:55,871] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user [POST]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 53, in post
    if args["client_id"] in marshal(Clients.query(Clients.id), Clients.response_fields):
TypeError: 'BaseQuery' object is not callable
[2019-12-17 20:23:55,871] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 5}, "response": {"message": "Internal Server Error"}}
[2019-12-17 20:24:28,400] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user [POST]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 53, in post
    if args["client_id"] in marshal(Clients.query(Clients.id), Clients.response_fields):
TypeError: 'BaseQuery' object is not callable
[2019-12-17 20:24:28,402] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 5}, "response": {"message": "Internal Server Error"}}
[2019-12-17 20:29:02,737] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "404 NOT FOUND", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 5}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 20:29:31,686] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "404 NOT FOUND", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 6}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 20:29:40,030] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /user [POST]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/user/resources.py", line 54, in post
    user = Users(args["name"], args["age"], args["sex"], args["client_id"])
TypeError: __init__() takes 4 positional arguments but 5 were given
[2019-12-17 20:29:40,033] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, "response": {"message": "Internal Server Error"}}
[2019-12-17 20:30:58,522] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, "response": {"created_at": "Tue, 17 Dec 2019 20:30:56 -0000", "updated_at": "Tue, 17 Dec 2019 20:30:56 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}}
[2019-12-17 20:31:09,972] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, "response": [{"created_at": "Tue, 17 Dec 2019 20:30:56 -0000", "updated_at": "Tue, 17 Dec 2019 20:30:56 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}]}
[2019-12-17 20:37:13,589] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, "response": [{"created_at": "Tue, 17 Dec 2019 20:30:56 -0000", "updated_at": "Tue, 17 Dec 2019 20:30:56 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}]}
[2019-12-17 20:37:36,748] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": {"created_at": "Tue, 17 Dec 2019 20:37:10 -0000", "updated_at": "Tue, 17 Dec 2019 20:37:10 -0000", "id": 2, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}}
[2019-12-17 20:37:47,832] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": [{"created_at": "Tue, 17 Dec 2019 20:30:56 -0000", "updated_at": "Tue, 17 Dec 2019 20:30:56 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, {"created_at": "Tue, 17 Dec 2019 20:37:10 -0000", "updated_at": "Tue, 17 Dec 2019 20:37:10 -0000", "id": 2, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}]}
[2019-12-17 20:38:03,842] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "DELETE", "code": "404 NOT FOUND", "request": null, "response": {"message": "NOT_FOUND"}}
[2019-12-17 20:38:07,274] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "DELETE", "code": "200 OK", "request": null, "response": {"message": "Deleted"}}
[2019-12-17 20:38:13,786] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": [{"created_at": "Tue, 17 Dec 2019 20:30:56 -0000", "updated_at": "Tue, 17 Dec 2019 20:30:56 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, {"created_at": "Tue, 17 Dec 2019 20:37:10 -0000", "updated_at": "Tue, 17 Dec 2019 20:37:10 -0000", "id": 2, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}]}
[2019-12-17 20:38:21,044] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": [{"created_at": "Tue, 17 Dec 2019 20:30:56 -0000", "updated_at": "Tue, 17 Dec 2019 20:30:56 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, {"created_at": "Tue, 17 Dec 2019 20:37:10 -0000", "updated_at": "Tue, 17 Dec 2019 20:37:10 -0000", "id": 2, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}]}
[2019-12-17 20:38:31,137] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": [{"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 2, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 3, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}, {"created_at": "Tue, 17 Dec 2019 19:44:08 -0000", "updated_at": "Tue, 17 Dec 2019 19:44:08 -0000", "id": 4, "client_key": "CLIENT01", "client_secret": "SECRET01", "status": true}]}
[2019-12-17 20:38:39,061] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": [{"created_at": "Tue, 17 Dec 2019 20:30:56 -0000", "updated_at": "Tue, 17 Dec 2019 20:30:56 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, {"created_at": "Tue, 17 Dec 2019 20:37:10 -0000", "updated_at": "Tue, 17 Dec 2019 20:37:10 -0000", "id": 2, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}]}
[2019-12-17 20:40:58,850] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": [{"created_at": "Tue, 17 Dec 2019 20:37:10 -0000", "updated_at": "Tue, 17 Dec 2019 20:37:10 -0000", "id": 2, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}]}
[2019-12-17 20:42:43,373] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": {"created_at": "Tue, 17 Dec 2019 20:37:10 -0000", "updated_at": "Tue, 17 Dec 2019 20:37:10 -0000", "id": 2, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}}
[2019-12-17 20:42:45,993] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": {"created_at": "Tue, 17 Dec 2019 20:30:56 -0000", "updated_at": "Tue, 17 Dec 2019 20:30:56 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}}
[2019-12-17 20:42:56,589] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "404 NOT FOUND", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 20:42:58,431] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": {"created_at": "Tue, 17 Dec 2019 20:37:10 -0000", "updated_at": "Tue, 17 Dec 2019 20:37:10 -0000", "id": 2, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}}
[2019-12-17 20:42:59,729] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "404 NOT FOUND", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": {"message": "NOT_FOUND"}}
[2019-12-17 21:12:50,265] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:12:54,231] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:13:07,404] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:13:11,833] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": null, "response": []}
[2019-12-17 21:13:25,203] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:16:15,966] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:17:39,649] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "PUT", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:17:55,347] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /client/1 [GET]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/client/resources.py", line 35, in get
    if qry.deleted_status is True or qry is None:
AttributeError: 'NoneType' object has no attribute 'deleted_status'
[2019-12-17 21:17:55,349] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "GET", "code": "500 INTERNAL SERVER ERROR", "request": null, "response": {"message": "Internal Server Error"}}
[2019-12-17 21:18:05,225] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:20:05,548] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:20:55,816] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:25:36,454] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:27:09,478] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:27:57,383] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"title": "Judul Buku Satu", "isbn": "1-234-5678-9101112-13", "writer": "Dr. Who"}, "response": []}
[2019-12-17 21:28:20,557] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"title": "Judul Buku Satu", "isbn": "1-234-5678-9101112-13", "writer": "Dr. Who"}, "response": {"created_at": "Tue, 17 Dec 2019 21:25:33 -0000", "updated_at": "Tue, 17 Dec 2019 21:25:33 -0000", "id": 1, "title": "Judul Buku Satu", "isbn": "1-234-5678-9101112-13", "writer": "Dr. Who"}}
[2019-12-17 21:28:50,092] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:29:42,004] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /client/6 [POST]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
TypeError: post() got an unexpected keyword argument 'id'
[2019-12-17 21:29:42,006] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "request": null, "response": {"message": "Internal Server Error"}}
[2019-12-17 21:30:23,907] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:31:08,353] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:31:24,855] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:32:03,673] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /client/8 [POST]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
TypeError: post() got an unexpected keyword argument 'id'
[2019-12-17 21:32:03,676] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "request": null, "response": {"message": "Internal Server Error"}}
[2019-12-17 21:32:32,888] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /client/8 [POST]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
TypeError: post() got an unexpected keyword argument 'id'
[2019-12-17 21:32:32,889] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "request": null, "response": {"message": "Internal Server Error"}}
[2019-12-17 21:32:48,138] {/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py:1891} ERROR - Exception on /client/8 [POST]
Traceback (most recent call last):
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta-18/Documents/Alterra/FlaskPractice/.flaskblog_venv/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
TypeError: post() got an unexpected keyword argument 'id'
[2019-12-17 21:32:48,140] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "request": null, "response": {"message": "Internal Server Error"}}
[2019-12-17 21:33:29,428] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:33:53,098] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:34:27,075] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_key": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:34:35,605] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_secret": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:36:26,246] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:43} ERROR - REQUEST_LOG	{"method": "POST", "code": "400 BAD REQUEST", "request": null, "response": {"message": {"client_secret": "Missing required parameter in the JSON body"}}}
[2019-12-17 21:38:40,799] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "akkk", "client_secret": "SECRET", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 21:38:30 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:30 -0000", "id": 1, "client_key": "akkk", "client_secret": "SECRET", "status": true}}
[2019-12-17 21:39:20,903] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "z", "client_secret": "SECRET", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 21:38:46 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:46 -0000", "id": 5, "client_key": "z", "client_secret": "SECRET", "status": true}}
[2019-12-17 21:40:37,928] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "z", "client_secret": "SECRET", "status": true}, "response": [{"created_at": "Tue, 17 Dec 2019 21:38:30 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:30 -0000", "id": 1, "client_key": "akkk", "client_secret": "SECRET", "status": true}, {"created_at": "Tue, 17 Dec 2019 21:38:46 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:46 -0000", "id": 5, "client_key": "z", "client_secret": "SECRET", "status": true}]}
[2019-12-17 21:40:49,767] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "zzzzz", "client_secret": "SECRET", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 6, "client_key": "zzzzz", "client_secret": "SECRET", "status": true}}
[2019-12-17 21:40:54,080] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "zzzzz", "client_secret": "SECRET", "status": true}, "response": [{"created_at": "Tue, 17 Dec 2019 21:38:30 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:30 -0000", "id": 1, "client_key": "akkk", "client_secret": "SECRET", "status": true}, {"created_at": "Tue, 17 Dec 2019 21:38:46 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:46 -0000", "id": 5, "client_key": "z", "client_secret": "SECRET", "status": true}, {"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 6, "client_key": "zzzzz", "client_secret": "SECRET", "status": true}]}
[2019-12-17 21:41:06,500] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 2}, "response": []}
[2019-12-17 21:41:16,485] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, "response": {"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}}
[2019-12-17 21:42:01,138] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"name": "Wildan", "age": 22, "sex": "Male", "client_id": 1}, "response": {"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 2, "name": "Wildan", "age": 22, "sex": "Male", "client_id": 1}}
[2019-12-17 21:42:11,669] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"name": "Wildan", "age": 22, "sex": "Male", "client_id": 1}, "response": [{"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 1, "name": "Wildan Firdaus", "age": 22, "sex": "Male", "client_id": 1}, {"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 2, "name": "Wildan", "age": 22, "sex": "Male", "client_id": 1}]}
[2019-12-17 21:42:22,770] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"name": "Wildan", "age": 22, "sex": "Male", "client_id": 5}, "response": {"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 3, "name": "Wildan", "age": 22, "sex": "Male", "client_id": 5}}
[2019-12-17 21:42:38,487] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "zzzzz", "client_secret": "SECRET", "status": true}, "response": [{"created_at": "Tue, 17 Dec 2019 21:38:30 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:30 -0000", "id": 1, "client_key": "akkk", "client_secret": "SECRET", "status": true}, {"created_at": "Tue, 17 Dec 2019 21:38:46 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:46 -0000", "id": 5, "client_key": "z", "client_secret": "SECRET", "status": true}, {"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 6, "client_key": "zzzzz", "client_secret": "SECRET", "status": true}]}
[2019-12-17 21:43:16,334] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "GET", "code": "200 OK", "request": {"client_key": "zzz", "client_secret": "SECRET", "status": true}, "response": [{"created_at": "Tue, 17 Dec 2019 21:38:30 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:30 -0000", "id": 1, "client_key": "akkk", "client_secret": "SECRET", "status": true}, {"created_at": "Tue, 17 Dec 2019 21:38:46 -0000", "updated_at": "Tue, 17 Dec 2019 21:38:46 -0000", "id": 5, "client_key": "z", "client_secret": "SECRET", "status": true}, {"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 6, "client_key": "zzzzz", "client_secret": "SECRET", "status": true}]}
[2019-12-17 21:43:23,885] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "zzz", "client_secret": "SECRET", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 21:39:43 -0000", "updated_at": "Tue, 17 Dec 2019 21:39:43 -0000", "id": 7, "client_key": "zzz", "client_secret": "SECRET", "status": true}}
[2019-12-17 21:45:11,955] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "zzz", "client_secret": "SECRET", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 21:45:05 -0000", "updated_at": "Tue, 17 Dec 2019 21:45:05 -0000", "id": 1, "client_key": "zzz", "client_secret": "SECRET", "status": true}}
[2019-12-17 21:45:18,718] {/home/alta-18/Documents/Alterra/FlaskPractice/blueprints/__init__.py:36} INFO - REQUEST_LOG	{"method": "POST", "code": "200 OK", "request": {"client_key": "zzzz", "client_secret": "SECRET", "status": true}, "response": {"created_at": "Tue, 17 Dec 2019 21:45:05 -0000", "updated_at": "Tue, 17 Dec 2019 21:45:05 -0000", "id": 2, "client_key": "zzzz", "client_secret": "SECRET", "status": true}}
